lib/components/helpers/pretty-text-input.js

```diff
@@ -283,7 +283,7 @@ export default React.createClass({
     return (
       <div onKeyDown={this.onKeyDown} className={cx({'pretty-text-wrapper': true, 'choices-open': this.state.isChoicesOpen})}
            onMouseEnter={this.switchToCodeMirror} onTouchStart={this.switchToCodeMirror}>
-        <div onClick={this.onFocusClick}>
+        <div className="sub-pretty-text-wrapper" onClick={this.onFocusClick}>
```

Comment:
Ok cool, thanks for pointing that out! Found the bug and fixed in https://github.com/zapier/zapier/pull/12121